### PR TITLE
Gadgets and QLIPHOLITH OVERLOAD

### PR DESCRIPTION
**Pull request by InsightfulParasite**
## Why It's Good For The Game

I felt like it would be nice to add some items that increased agent and clerk survival rate so i put in a few devices that can hinder the movement of abnormalities(and only abnormalities for now) and give people a second chance when they go insane due to non lethal sources.

## Changelog
:cl:
add: Qlipholith overload status effect and the qliphoth field projector, along with sanity insurance status effect and moonstones.
add: abnormality detector
add: prototype shields and the manager bullet.
add: restingpot that gives you lumps when fed
add: violet mass, weird orb that (admin spawned) falls from violet ordeals.
add: surplus item sprites.
add: department nav beacons in waypoints
add: forgot i put in sprites for a wearable sweepermask and LC13 userface icons. Though the prudence icon is used for sanity insurance.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/gutlunch.dm [hunk 1]
```diff
@@ -43,7 +43,7 @@
 	animal_species = /mob/living/simple_animal/hostile/asteroid/gutlunch
 	childtype = list(/mob/living/simple_animal/hostile/asteroid/gutlunch/grublunch = 100)
 
-	wanted_objects = list(/obj/effect/decal/cleanable/xenoblood/xgibs, /obj/effect/decal/cleanable/blood/gibs/, /obj/item/organ)
+	wanted_objects = list(/obj/effect/decal/cleanable/xenoblood/xgibs, /obj/effect/decal/cleanable/blood/gibs/, /obj/item/organ, /obj/item/bodypart/head, /obj/item/bodypart/head)
```
Comment (InsightfulParasite) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/gutlunch.dm:46]: This was a accident i need to fix this.

**Review by dsmith328 — COMMENTED** [review]
I'd like @EgorDinamit to check the code on the Qliphoth device.

**Comment by dsmith328** [comment]
Cleared up Icon conflicts and squashed Commits. Let me know when you're satisfied with the amount of content added and I'll probably just straight up merge this.

**Review by dsmith328 — APPROVED** [review]

**Comment by InsightfulParasite** [comment]
I dont know if i even want to touch these commits after what it took to remove the edits made to DMI files.

File: ModularTegustation/tegu_items/lc13_uniqueitems.dm [hunk 2]
```diff
@@ -0,0 +1,559 @@
...
+/obj/item/moonlightstone/dropped(mob/living/carbon/human/user)
+	..()
+	user.remove_status_effect(STATUS_EFFECT_SANITYINSURANCE)
+
+/datum/status_effect/sanity_insurance
+	id = "sanity insurance"
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = /atom/movable/screen/alert/status_effect/sanity_insurance
+
+/datum/status_effect/sanity_insurance/tick() //i dont think there is any way to avoid using tick() Nevermind the managerbullet can use signals that are given out by certain events.
```
Comment (EgorDinamit) [inline ModularTegustation/tegu_items/lc13_uniqueitems.dm:169]: Use signal instead.

File: ModularTegustation/tegu_items/lc13_uniqueitems.dm [hunk 3]
```diff
@@ -0,0 +1,559 @@
...
+	var/faltering = 0
+
+/datum/status_effect/interventionshield/on_apply()
+	. = ..()
+	owner.add_overlay(statuseffectvisual)
+	var/mob/living/carbon/human/L = owner
+	switch(respectivedamage)
+		if(RED_DAMAGE)
+			inherentarmorcheck = L.physiology.red_mod
+			L.physiology.red_mod *= 0.0001
```
Comment (EgorDinamit) [inline ModularTegustation/tegu_items/lc13_uniqueitems.dm:231]: This is certainly not how we should do it...

Comment (InsightfulParasite) [reply ModularTegustation/tegu_items/lc13_uniqueitems.dm:231]: Should i remove it? From my tests it seems to work and fear damage bypasses it completely

Comment (InsightfulParasite) [reply ModularTegustation/tegu_items/lc13_uniqueitems.dm:236]: This is also how life for a daredevil also handles it.

**Pull request closed without merging**